web/docs/language/features.md

```diff
@@ -728,7 +728,7 @@ export const signUp = async (args, context) => {
     // Your custom code before sign-up.
     // ...
     const newUser = context.entities.User.create({
-        data: { email: 'some@email.com', password: 'this will be hashed!' }
+        data: { username: 'someone', password: 'this will be hashed!' }
```

Comment:
Let me know if we want to take this opportunity to spice this example username up any :D

Reply:
We definitely should, but I can't think of anything that satisfies both of the following conditions:
1. It's funny
2. It wouldn't get us cancelled

Reply:
Some ideas:
- waspeteer
- buzzybee
- waspgoesbzz
- wasper
- coolperson

Ok none of these is very funny hm :D.

Reply:
Ah, maybe `waspeteer`... I'll go with that one since it is at least Wasp-related, thnx 👍🏻